rubyUtils.gradle

```
@@ -183,7 +183,7 @@ task downloadJRuby(type: Download) {
     description "Download JRuby artifact from this specific URL: ${jRubyURL}"
     src jRubyURL
     onlyIfNewer true
-    inputs.file("${projectDir}/versions.yml")
+    inputs.file("${versionsPath}/versions.yml")
```

versionsPath already contains the yaml file, so we can just do:

```suggestion
    inputs.file(versionsPath)
```

---

Thanks, @jsvd and @andsel. You're right about the path and I've updated the PR accordingly.